packages/react-components/src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import '../../dist/theme.css';
```

Comment:
remove theme.css line.
import theme.css in your test application

Reply:
Removed!